_python/main/templates/casebook_edit.html

```diff
@@ -13,7 +13,11 @@
 {% endblock %}
 
 {% block mainContent %}
-  <header class="casebook-draft casebook" data-casebook-id="{{ casebook.id }}">
+  <header class="casebook-draft casebook" data-casebook-id="{{ casebook.id }}"
+          data-section-id=""
+          data-section-ordinals=""
+          data-resource-id=""
+  >
```

Comment:
Interesting. These empty data attributes are used to make the editable TOC work?